### ISSUE TITLE
Tree View selected state not announced

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3571]
## Describe the bug

The selected state of treenode is not announced by screen readers. Currently, when a treenode is selected, a background color of #d9e4ea is applied. But blind screen users cannot see this background color and do not hear any feedback telling the a given treenode has been selected.

## How to reproduce

https://stackblitz.com/github/clane/clarityTest2/tree/master/clarity-test-app

- Using a screen reader open the Tree View - Icons demo
- "Highlight" one of the tree nodes by using the Tab key to focus it and then pressing either the Enter or the Spacebar keys
- Notice that the screen reader does not announce any information regarding the selected state of the treenode

## Expected behavior

Use aria-selected to convey the selected state of a treenode as described at https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by hippee-lee** [comment]
This one seems like when a `tree-node-content tree-node-button` is active it also needs to add aria-selected.

**Comment by chlane** [comment]
Looking at https://clarity.design/documentation/tree-view#basic-tree we are still not using aria-selected to match the application of the active class below. 

.clr-treenode-link.active {
    background: #d8e3e9;
    background: var(--clr-tree-link-selection-color,#d8e3e9);
    color: #000;
    color: var(--clr-tree-node-caret-link-hover-color,#000);
}
